retworkx/__init__.py

```diff
@@ -1303,6 +1303,8 @@ def networkx_converter(graph):
         independently.
 
     :param networkx.Graph graph: The networkx graph to convert.
+    :param bool keep_attributes: If `True`, add networkx attributes to retworkx
+        object.
```

Comment:
Could you add a bit to this about what the data payloads for the nodes would look like when this is set to true. Maybe reword this to be something like?

```suggestion
    :param bool keep_attributes: If `True`, add networkx node attributes to the data
        payload to the nodes in the output retworkx graph. When set to ``True`` the
        node data payloads in the output retworkx graph object will be dictionaries
        with the node attributes from the input networkx graph where the
        ``"__networkx_name__"`` key contains the node object from the input networkx
        graph.
```